### ISSUE TITLE
[SCREENREADER]: Should update focus management to H1 on apply for education benefits selection pages

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#7519]
**⚠️Should** update. This is a best practice that has evolved since it was originally put in place.

## Description

While I was reviewing a separate item for the STEM update, I noticed Chrome + NVDA wasn't managing focus correctly on the Apply for Benefits introductory page. I was on Form 1995 at the time, but I'm almost positive every education form uses this same CTA intake.

This was causing confusion because I couldn't tell the new page had loaded. I'd like to set focus on the H1 instead of our breadcrumb, as an updated best practice. Screenshot attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

- [ ] Screen readers announce the H1 properly on page load
- [ ] Confirm all education forms (1990, 1990e, 1990n, 1995, etc) use the same CTA page and have the same behavior

## Environment

* Windows 10
* Chrome latest
* NVDA
* https://staging.va.gov/education/apply-for-education-benefits/application/1995/introduction

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs

![Screen Shot 2020-04-01 at 4 28 24 PM](https://user-images.githubusercontent.com/934879/78189312-67b28f00-7437-11ea-8785-af3a9909e608.png)

**Comment by joshkimux** [comment]
@artsymartha68 this ticket does not belong to the benefits team anymore I believe, could we re-route this to the education folk?

**Comment by jeana-adhoc** [comment]
Navigating from https://staging.va.gov/education/how-to-apply/ and into the education forms from the deprecated wizard, loads the pages with no focus management, and the user starts at the top of the page. In NVDA, it does announce the new page as being loaded. 

Closing this ticket as resolved.
